test/extensions/filters/http/aws_lambda/aws_lambda_filter_test.cc

```diff
@@ -63,7 +86,7 @@ class AwsLambdaFilterTest : public ::testing::Test {
  * Requests that are _not_ header only, should result in StopIteration.
  */
 TEST_F(AwsLambdaFilterTest, DecodingHeaderStopIteration) {
-  setupFilter({arn_, InvocationMode::Synchronous, true /*passthrough*/, ""});
+  setupDownstreamFilter(InvocationMode::Synchronous, true, "");
```

Comment:
can we keep `/*passthrough*/` block comment for readability?

Reply:
sure